content/pages/support-groups.md

```diff
@@ -1,38 +1,42 @@
 ---
 title: Support Groups
 slug: support-groups
-description: A list of online and in-person support groups for transgender and non-binary Alabamians
+description: A collection of support groups for transgender, nonbinary, and gender nonconforming Alabamians
```

Comment:
It's not that I don't think GNC people are cool and awesome. But cis GNC people are not allowed in most of these groups so its probably inaccurate to include here.

```suggestion
description: A collection of support groups for transgender and nonbinary Alabamians
```

Reply:
oh the wording used elsewhere is also a fine option for this too